src/keybindings.c

```diff
@@ -1375,6 +1375,11 @@ static gboolean on_key_press_event(GtkWidget *widget, GdkEventKey *ev, gpointer
 	if (keyval >= GDK_KP_Space && keyval < GDK_KP_Equal)
 		keyval = key_kp_translate(keyval);
 
+	if (keyval == GDK_KEY_Escape)
+	{
+		/* close search bar */
+		ui_emit_button_close_in_search_bar_clicked_if_visible(doc);
+	}
```

Comment:
Here we should rather have a keybinding. I used to map escape to back, in which case, I wouldn't want the search bar to be closed (if the focus is on the editor).